azerothcore/azerothcore-wotlk
# fix(DB/creature_loot_template,conditions): Cracked Silithid Carapace (5877)

## Changes Proposed:
Drop item **Cracked Silithid Carapace (5877)** at 100% only while on the quest **The Swarm Grows (1147)**


## Issues Addressed:
- Closes #6280
- Closes https://github.com/chromiecraft/chromiecraft/issues/815


## SOURCE:
https://wowpedia.fandom.com/wiki/Cracked_Silithid_Carapace?oldid=2026412
> This item drops from Silithid Searchers, Silithid Hive Drones, and Silithid Invaders at the Rustmaul Digsite in the Shimmering Flats, Thousand Needles. If you are eligible for the drop, it will drop from the first Silithid you kill here.

https://www.wowhead.com/item=5877/cracked-silithid-carapace#comments:id=243225
> Actually, I think you need to have the "The Swarm Grows" quest to find this drop. I killed 100+ of them and didn't find any but as soon as I got the quest I found the drop. On the first kill.


## Tests Performed:
- SQL statements successfully executed
- Tested in game


## How to Test the Changes:
- Kill and loot **Silithid Searcher, Ravager, Invader, Hive Drone**  at the **Rustmaul Digsite** in the **Shimmering Flats / Thousand Needles** while **not** on the quest **The Swarm Grows (1147)** and observe that the quest item does not drop
- `.add quest 1147`
- Kill some more and observe that the quest item is dropped on first kill but no longer after accepting the quest **Parts of the Swarm (1148)** from it.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
